Add an Introduction

Sonna opened the pull request:
Quotes:

> Create a new file called introduction.md introducing yourself to the
> dojo. Make sure this file contains an introduction, then add, commit
> and push your change to the fork.
> 
> -- CodeSchool

PeterBell committed:
Merge pull request #2 from githubteacher/master

Most Senseis should be respected

Commit:
Initial kill list

PeterBell committed:
Merge pull request #3 from deadlyvipers/kill_list

Initial kill list

Commit:
We dont need to kill Budd

Sonna committed:
Add an Introduction to intro myself

Quotes:
> Create a new file called introduction.md introducing yourself to the
> dojo. Make sure this file contains an introduction, then add, commit
> and push your change to the fork.
>
> -- CodeSchool

codeschool-kiddo commented on introduction.md:
```diff
@@ -0,0 +1,3 @@
+# Introduction
```
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

Sonna committed:
Update Introduction about CodeSchool

Quotes:
> Looks good! Could you also please mention your favorite Code School
> path in your introduction? This could be Ruby, JavaScript, HTML & CSS,
> iOS or Electives.
>
> -- CodeSchool

Sonna committed:
Update introduction with deadly skills

Quotes:
> On your "deadly_skills" branch, open up your introduction file and
> modify it to include a list of your deadly skills (for example: Git,
> JavaScript, Ruby). Since this file is Markdown, you can specify these
> one per line starting the line with a * to make them into a list.
>
> Add, commit and push your changes up to GitHub. Be sure to push to the
> remote branch with the same name as your local branch – deadly_skills.
>
> -- CodeSchool

Sonna committed:
Update introduction with deadly skills

Quotes:
> On your "deadly_skills" branch, open up your introduction file and
> modify it to include a list of your deadly skills (for example: Git,
> JavaScript, Ruby). Since this file is Markdown, you can specify these
> one per line starting the line with a * to make them into a list.
>
> Add, commit and push your changes up to GitHub. Be sure to push to the
> remote branch with the same name as your local branch – deadly_skills.
>
> -- CodeSchool

> Since you're becoming a GitHub master, could you also add "Killing
> history using git rebase" as one of your deadly skills?
>
> -- CodeSchool (GitHub)

codeschool-kiddo committed:
Added new rule about coffee

Sonna committed:
Update Dojo Rules

Quotes:
> OK, our files are all committed and pushed up to your "dojo_rules"
> repo on the "new_rules" branch. Switch back to the master branch and
> pull down the latest changes and switch over to the "new_rules"
> branch.
>
> If you take a look at the "dojo_rules.md" file, you'll notice we were
> a bit careless in our new rule – misspelling "coffee" and capitalizing
> "dojo" when it should be lowercase. Go ahead and fix both of these
> errors on the branch and push your changes up to GitHub also on the
> "new_rules" branch.
>
> Your new commit should also now show up in the pull request we
> created. Once you've verified all of this, let us know by clicking
> "Submit Answer".
>
> -- CodeSchool

Sonna committed:
Merge pull request #2 from Sonna/new_rules

Too many people are spilling coffee

Sonna committed:
Merge branch 'deadly_skills'

* deadly_skills:
  Update introduction with deadly skills

Sonna committed:
Update Kill List

Quotes:
> Now that you've created the release branch, replace the assassin names
> in kill_list.md with the following bad coding practices:
>
> * Unused code
> * Functions with side effects
> * Unformatted code
> * Repeated code blocks
> * Bad variable names
>
> -- CodeSchool

Sonna committed:
Merge branch 'release_branch_1.0'

* release_branch_1.0:
  Update Kill List
  We dont need to kill Budd
  Initial kill list

Sonna committed:
Update Kill list with personal grievance

Quotes:
> To add a bit of a personal touch, add a programmer grievance of your
> own to the "kill_list.md" file.
>
> -- CodeSchool

Fixes #3 "Contribute to the Kill List"

Sonna committed:
Merge pull request #4 from Sonna/kill_list

Update Kill list with personal grievance

Sonna committed:
Update README.md

Quotes:
> Our "README.md" file on the master branch of our "dojo_rules" repository isn't very helpful. Update "README.md" to mention that "All members should read the rules".
>
> Also include a link back to the deadlyvipers organization which started it all ("https://github.com/deadlyvipers").
>
> Feel free to include any other GitHub Flavored Markdown as well.
>
> -- CodeSchool

Sonna commented:
Closing this Open Pull Request, since nothing has happened to it in 4 years, and would personally like to clear [my "created" tab on the Pull Requests page](
https://github.com/pulls?q=is%3Aopen+is%3Apr+archived%3Afalse+author%3ASonna)
